test/f90_correct/src/import_mod_from_user.f90

```diff
@@ -0,0 +1,16 @@
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+!
+! Test for the import module from the correct path.
+
+module import_mod_from_user
+contains
+  subroutine sub()
+     use import_source2
+  end
+end
+
+  print *, "PASS"
+end
```

Comment:
This looks like a dangling `end`.

Reply:
Right. It is for testing for execution. Is there any other way in classic-flang test tools?

Reply:
Fixed. Adding the program name truly looks more beautiful.